circle.yml

```diff
@@ -0,0 +1,17 @@
+machine:
+  java:
+    version: oraclejdk8
```

Comment:
Hm.. doesn't look like circle [has openjdk8](https://circleci.com/docs/1.0/build-image-precise/#java), so if we want java 8, we have to go with oracle.